src/actions/albums.js

```diff
@@ -45,6 +46,46 @@ export const fetchAlbums = (mangoIndex) => {
   }
 }
 
+// Returns albums photos informations from the album ID
+export const fetchAlbumPhotosStatsById = (albumId) => {
```

Comment:
I usually prefer using `then().catch()` because it's more explicit and accurate about errors handling and the way that asynchronous feature works. But here, I agree with you, it's hard to read and a `try catch` would be better. I will do that, thanks 👍